app/scripts/controllers/landing.js

```diff
@@ -44,6 +45,11 @@ angular.module('confRegistrationWebApp')
         };
         $scope.searchComplete = val;
         $scope.searchResults = response.data;
+
+        analytics.digitalData.searchFilter = '';
```

Comment:
@adammeyer @brenton05 What is supposed to be here? Is it necessary to send this with an empty string? Did this just not get finished in the other PR?